packages/paste-core/components/avatar/src/index.tsx

```diff
@@ -48,23 +48,28 @@ AvatarContents.displayName = 'AvatarContents';
 const colorVariants: Record<ColorVariants, BoxStyleProps> = {
   default: {
     backgroundColor: 'colorBackgroundUser',
-    color: 'colorText',
+    color: 'colorTextUser',
+    borderColor: 'colorBorderUser',
```

Comment:
todo: Avatars need to use shadow borders to align in height with other components (buttons, etc).

See work breakdown doc: https://docs.google.com/document/d/1YDt4_TEcBNT0RqQwAVL_APMkDnMRf5YAy4eW9UttXxQ/edit#bookmark=id.ju18wvpkllnr